resources/js/controllers/components/menu_controller.js

```diff
@@ -278,6 +278,9 @@ export default class extends Controller {
      * @returns {*}
      */
     getUri(path = '') {
+        if (path == '') {
```

Comment:
What do you think of such a record?
```javascript
path = path === '' ? `/press/menu` : `/press/menu/${path}`;
return platform.prefix(path);
```

Reply:
of course, that is cool.
I will change.